### ISSUE TITLE
Old pre-built binary versions

### ISSUE THREAD

**Issue by hyperknot** [pelias/pbf2json#89]
### Use-cases
The user should be able to figure out:
- How to get the latest pre-built binary
- What is the version number of a given pre-built binary

### Attempted Solutions
I looked at GitHub / master branch, where:
- releases say latest version is 6.5.0
- the build folder contains files from 10 month ago, so it cannot be 6.5.0, but not even 6.4.0. Probably they are from 6.3.0
- I tried to run --version and --help on them, but it was not possible to see the version in any way
- I tried downloading from npm, again, I had no idea what binary version is included there.

### Proposal
- Document the above points
- Add a --version switch so that it's clear what version is a given pre-built binary.

**Comment by hyperknot** [comment]
Update: I did a binary compare on npm's build and github master's build, they are the same file. So it seems that both versions are really old, around 6.3.0.

**Comment by missinglink** [comment]
Yes, I think the confusion is that the Go code and the JavaScript code are developed independently.

We use `semantic-release` for our version of `npm` so the minor version has incremented twice since the latest Go code changes in https://github.com/pelias/pbf2json/pull/82.

Regarding your use-case, the latest version of the `npm` module will always include the latest version of the go code and the latest binary published on GitHub will always include the latest code.

Adding a version switch to the ho binary would be a nice addition.
